caddyconfig/httpcaddyfile/builtins.go

```diff
@@ -59,6 +59,11 @@ func parseBind(h Helper) ([]ConfigValue, error) {
 //         protocols <min> [<max>]
 //         ciphers   <cipher_suites...>
 //         curves    <curves...>
+//         clients	 {
+//                     mode	              [request|require|verify_if_given|require_and_verify]
+//                     trusted_ca_certs   <trusted_ca_certs...>
+//                     trusted_leaf_certs <trusted_leaf_certs...>
+//					 }
```

Comment:
Just some whitespace cleanup on this comment (tabs -> spaces, indentation fixes)

```suggestion
//         clients {
//             mode	              [request|require|verify_if_given|require_and_verify]
//             trusted_ca_certs   <trusted_ca_certs...>
//             trusted_leaf_certs <trusted_leaf_certs...>
//         }
```